lib/test_suite.py

```diff
@@ -1,4 +1,16 @@
-import ConfigParser
+from __future__ import print_function
+from __future__ import absolute_import
+
+from six.moves import configparser
```

Comment:
Well, I don't mind regarding solution, both are ok for me. Rewrote a patch and new version uses different imports for Python 2 and Python 3.

```python
--- a/lib/test_suite.py
+++ b/lib/test_suite.py
@@ -1,4 +1,10 @@
-import ConfigParser
+try:
+    # Python 3
+    import configparser
+except ImportError:
+    # Python 2
+    import ConfigParser as configparser
+
 import json
 import os
 import re
@@ -90,7 +96,7 @@ class TestSuite:
             raise RuntimeError("Suite %s doesn't exist" % repr(suite_path))
 
         # read the suite config
-        config = ConfigParser.ConfigParser()
+        config = configparser.ConfigParser()
         config.read(os.path.join(suite_path, "suite.ini"))
         self.ini.update(dict(config.items("default")))
         self.ini.update(self.args.__dict__)
```
Regarding using `six`: yes, probably it's good to drop it from dependencies list, but benefits are not clear for me. Anyway we have other dependencies in `requirements.txt`...

Reply:
BTW below you highlighted a good reason to reduce dependencies: `Since we can easily eliminate the dependency I hope we'll have less problems with RPM / Deb dependencies in tarantool and a few connectors and modules that also use test-run.`.